src/object/cli_obj.c

```diff
@@ -1253,13 +1257,20 @@ obj_iod_valid(unsigned int nr, daos_iod_t *iods, bool update)
 	return true;
 }
 
+static daos_epoch_t
```

Comment:
maybe make this inline

Reply:
right, I think the compiler would inline it automatically when needed.